inc/class-cachify.php

```diff
@@ -769,6 +825,16 @@ public static function process_flush_request( $data ) {
 				);
 			}
 		}
+
+		/* Reschedule HDD Cache Cron */
+		if ( self::METHOD_HDD == self::$options['use_apc'] ) {
+			$timestamp = wp_next_scheduled( 'hdd_cache_cron' );
+			if ( false != $timestamp ) {
```

Comment:
same, !== instead of !=

Reply:
Is included in "Make requested changes to checks" and found the same issue in line 300